File: URI/Lua/ValidacaoDeNotas.lua
media = 0
qte = 0
while true do
    nota = io.read("*number")
    if 0 <= nota and nota <= 10 then
        media = media + nota
        qte = qte + 1
    else
        print("nota invalida")
    end

    if qte == 2 then
        break
    end
end
print(string.format( "media = %.2f",media/2 ))

File: URI/Python/ValidacaoDeNotas.py
media = 0
qte = 0
while True:
    nota = float(input())
    if 0 <= nota <= 10:
        media += nota
        qte+=1
    else:
        print("nota invalida")
    
    if qte == 2:
        break
print("media = {:.2f}".format(media/2))
